src/platform/qt/src/config.hpp

```diff
@@ -81,6 +83,15 @@ struct QtConfig final : nba::PlatformConfig {
   }
 
   void Save() {
+    auto config_dir = fs::path(config_path).parent_path();
+    if (!fs::exists(config_dir)) {
+        try {
+            fs::create_directories(config_dir);
+        } catch (const std::exception& ex) {
+            qDebug() << "HHH";
+        }
+    }

```

Comment:
Please remove the try-catch or implement a path to reasonably handle the error. Also adjust the indentation please (we use two spaces to indent)